### DISCUSSION TITLE
Centering Text

### DISCUSSION THREAD

**Discussion by Joseph-Zaki** [retypeapp/retype#39] [Q&A] [answered]
Is there a method of centering text in Retype? Using HTMLs text-align attribute has not worked.

**Comment by geoffreymcgill** [comment, marked as answer]
Unfortunately, Markdown does not provide a text centering option, but adding raw HTML to a document is supported. 

For example, using the `<center>`, `<p>`, or `<div>` tags can work:

```md
# Center content sample

This is a normal markdown paragraph.

<center>Some centered content.</center>

<p style="text-align: center;">More centered content.</p>

<div style="text-align:center">One more center content option.</div>

This is a normal markdown paragraph.
```

If you add HTML to the document, the content within the HTML will not be parsed by Retype. If you have formatting within those HTML content elements, you'll need to manually format using HTML.

Hope this helps.

**Reply by Joseph-Zaki** [reply]
Thank you! Turns out the text-align attribute does work, and I need to brush up on my HTML.
